test/unit/component/parser/json/BpmnJsonParser.process.test.ts

```diff
@@ -434,9 +433,6 @@ describe('parse bpmn as json for process/pool', () => {
         height: 430,
       },
     });
-
-    // Check detected participants during json parsing
-    expect(findProcessRefParticipant('Participant_2')).toBeUndefined;
```

Comment:
I think we have to keep this checking.

Reply:
After reading the test fully, I think we cover this case, because we verfiy that there is only one Pool and it's not 'Participant_2' 😄

Reply:
Yes, that what I had in mind with 5da8e139d804be92fcca4541ea816a7e4375a357.
We want to test the parsing result not the internal implementation.

I wrote the initial test and this expectation, I don't remember why I added it :smile: (at that time, we were using the former json parsing, so maybe I was necessary...)